### DISCUSSION TITLE
Possibility of using T-Display s3

### DISCUSSION THREAD

**Discussion by wilu92PL** [alexanderlavrushko/BLE-HUD-navigation-ESP32#29] [General]
Good morning :) I really like this project I would like to use it for trips on my motorbike. 
Will it be possible to use the T- Display s3? And if modifications are required will it be difficult? I am a beginner in this.

**Comment by alexanderlavrushko** [comment]
Hello @wilu92PL , I don't have T-Display S3 module and don't know how to program it, and I don't plan to purchase it (at least for now). So unfortunately I cannot say how difficult it is to adjust the project for it.

**Comment by pigro141** [comment]
Hi @wilu92PL and @alexanderlavrushko,
I wanted to share that I’ve created my own modified version of the project specifically for the T-Display S3.
It’s based on Alexander’s original repo but includes some fixes and changes to support this display properly.
The modifications are a bit advanced, but if @wilu92PL wants to use the T-Display S3, they can follow my project, copy and paste the necessary code, and follow the instructions in my repo:
https://github.com/pigro141/BLE-HUD-navigation-ESP32-LilygoTDisplayS3
Thanks to Alexander for the original work, and I hope my version can help anyone wanting to use the S3!
